utils.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+import awswrangler as wr
+import boto3
+import pandas as pd
+import requests
+
+logging.basicConfig(format='%(asctime)s %(levelname)s:%(name)s:%(message)s')
+logging.getLogger().setLevel(20)
+
+client = boto3.client('s3')
+logging.info("s3 client initiated")
+client.create_bucket(Bucket='ayodeji-data-ingestion-bucket')
```

Comment:
you can't do this, use Terraform to provision your s3 bucket

Reply:
Alright, sir

Reply:
> you can't do this, use Terraform to provision your s3 bucket

I have now provisioned with s3 terraform and tested 20 profiles, sir